tests/lib/linter/linter.js

```diff
@@ -6757,7 +6761,7 @@ var a = "test2";
                 create: context => ({
                     Program(ast) {
                         receivedFilenames.push(context.filename);
-                        receivedPhysicalFilenames.push(context.getPhysicalFilename());
+                        receivedPhysicalFilenames.push(context.physicalFilename);
```

Comment:
It would be good to add `assert.strictEqual(context.getFilename(), context.filename)` and `assert.strictEqual(context.getPhysicalFilename(), context.physicalFilename)` here.

Reply:
@mdjermanovic Added 👍🏻